README.rst

```diff
@@ -18,9 +26,104 @@ There is no released code API.
   Sometimes, it may not work at all.
 
 Each official release is tagged in the repository. The `change
-history`_ describes every version.
+history` _describes every version.
+
+ROS usage
+---------
```

Comment:
I'd be glad to get the docs to docs.ros.org. I just don't have any experience getting it there (and possibly testing before rolling out the public version of the documentation).

I'd appreciate your help ;)